openshift-start-configured-cluster

```diff
@@ -43,6 +43,17 @@ for i in openshift-apiserver openshift-controller-manager; do
     sed -i 's|webhookTokenAuthenticators: null|webhookTokenAuthenticators:\n  - configFile: "webhook.yaml"|' $OC_CONFIG/$i/master-config.yaml
 done
 
+echo "Generating Keycloak client cert"
+# This need to happen before the oc cluster
+openssl genrsa -out $OC_CONFIG/openshift-apiserver/keycloak-client.key 2048
+openssl req -new -key $OC_CONFIG/openshift-apiserver/keycloak-client.key -out $OC_CONFIG/openshift-apiserver/keycloak-client.csr -subj "/CN=api.keycloak.localhost"
+openssl x509 -req -in $OC_CONFIG/openshift-apiserver/keycloak-client.csr -CA $OC_CONFIG/openshift-apiserver/ca.crt -CAkey $OC_CONFIG/openshift-apiserver/ca.key -CAcreateserial -out $OC_CONFIG/openshift-apiserver/keycloak-client.crt -days 1024 -sha256
+rm -rf $OC_CONFIG/openshift-apiserver/keycloak-client.csr
+cp $OC_CONFIG/openshift-apiserver/keycloak-client.crt $OC_CONFIG/kube-apiserver
+cp $OC_CONFIG/openshift-apiserver/keycloak-client.crt $OC_CONFIG/openshift-controller-manager
+cp $OC_CONFIG/openshift-apiserver/keycloak-client.key $OC_CONFIG/kube-apiserver
+cp $OC_CONFIG/openshift-apiserver/keycloak-client.key $OC_CONFIG/openshift-controller-manager
+
```

Comment:
why are you copying this file in multiple places when the webhook.yaml file points to a fixed path on disk ?

Reply:
I noticed that all certificates are present in all 3 directories (`openshift-controller-manager`, `kube-apiserver` and `openshift-apiserver`). I thought I should do the same. If I was mistaken, please tell me what exactly should I do (I assume, I should copy the key and cert to `kube-apiserver`).